src/pages/Entries/EntryCard.tsx

```diff
@@ -182,9 +180,14 @@ export default function EntryCard({
         id="delete-confirm-modal"
         title="Deleting happiness"
         // fix comment form
-        body={`You are deleting happiness for ${parseYYYYmmddFormat(
+        body={`You are deleting happiness for ${dateFromStr(
```

Comment:
why did you change out the date function here? just curious

Reply:
i think i just did that to be consistent with the other change I made on the same file